From 34713b3d399c935f5f46b82cf16197aa9940344e Mon Sep 17 00:00:00 2001
From: idusergod <artem.martyanov06@gmail.com>
Date: Thu, 6 Aug 2026 21:36:02 +0300
Subject: [PATCH 11/56] arm64: configs: add
 kaanapali-oneplus-infiniti_defconfig fragment

Config fragment for OnePlus 15 on top of the arm64 defconfig. Apply with
merge_config.sh -m, then olddefconfig.

Signed-off-by: idusergod <artem.martyanov06@gmail.com>
y705-source: 0005-arm64-configs-add-kaanapali-oneplus-infiniti_defconf.patch
---
 .../kaanapali-oneplus-infiniti_defconfig      | 873 ++++++++++++++++++
 1 file changed, 873 insertions(+)
 create mode 100644 arch/arm64/configs/kaanapali-oneplus-infiniti_defconfig

diff --git a/arch/arm64/configs/kaanapali-oneplus-infiniti_defconfig b/arch/arm64/configs/kaanapali-oneplus-infiniti_defconfig
new file mode 100644
index 000000000..096d15442
--- /dev/null
+++ b/arch/arm64/configs/kaanapali-oneplus-infiniti_defconfig
@@ -0,0 +1,873 @@
+# SPDX-License-Identifier: GPL-2.0
+#
+# Fragment for OnePlus 15 (kaanapali / canoe).
+# Usage:
+#   make ARCH=arm64 defconfig
+#   ./scripts/kconfig/merge_config.sh -m .config \
+#       arch/arm64/configs/kaanapali-oneplus-infiniti_defconfig
+#   make ARCH=arm64 olddefconfig
+#
+
+# CONFIG_AMPERE_ERRATUM_AC03_CPU_38 is not set
+# CONFIG_AMPERE_ERRATUM_AC04_CPU_23 is not set
+CONFIG_ANDROID_BINDER_DEVICES="binder,hwbinder,vndbinder"
+CONFIG_ANDROID_BINDER_IPC=y
+# CONFIG_AQUANTIA_PHY is not set
+# CONFIG_ARCH_ACTIONS is not set
+# CONFIG_ARCH_AIROHA is not set
+# CONFIG_ARCH_ALPINE is not set
+# CONFIG_ARCH_APPLE is not set
+# CONFIG_ARCH_ASPEED is not set
+# CONFIG_ARCH_AXIADO is not set
+# CONFIG_ARCH_BCM is not set
+# CONFIG_ARCH_BERLIN is not set
+# CONFIG_ARCH_BLAIZE is not set
+# CONFIG_ARCH_BST is not set
+# CONFIG_ARCH_CIX is not set
+# CONFIG_ARCH_EXYNOS is not set
+CONFIG_ARCH_HAS_SUBPAGE_FAULTS=y
+CONFIG_ARCH_HAS_ZONE_DMA_SET=y
+# CONFIG_ARCH_HISI is not set
+# CONFIG_ARCH_INTEL_SOCFPGA is not set
+# CONFIG_ARCH_K3 is not set
+# CONFIG_ARCH_KEEMBAY is not set
+# CONFIG_ARCH_LG1K is not set
+# CONFIG_ARCH_MA35 is not set
+# CONFIG_ARCH_MEDIATEK is not set
+# CONFIG_ARCH_MESON is not set
+# CONFIG_ARCH_MICROCHIP is not set
+# CONFIG_ARCH_MVEBU is not set
+# CONFIG_ARCH_NPCM is not set
+# CONFIG_ARCH_NXP is not set
+# CONFIG_ARCH_REALTEK is not set
+# CONFIG_ARCH_RENESAS is not set
+# CONFIG_ARCH_ROCKCHIP is not set
+# CONFIG_ARCH_SEATTLE is not set
+# CONFIG_ARCH_SOPHGO is not set
+# CONFIG_ARCH_SPRD is not set
+# CONFIG_ARCH_STM32 is not set
+# CONFIG_ARCH_SUNXI is not set
+CONFIG_ARCH_SUPPORTS_SHADOW_CALL_STACK=y
+# CONFIG_ARCH_SYNQUACER is not set
+# CONFIG_ARCH_TEGRA is not set
+# CONFIG_ARCH_THUNDER is not set
+# CONFIG_ARCH_THUNDER2 is not set
+# CONFIG_ARCH_UNIPHIER is not set
+CONFIG_ARCH_USES_PG_ARCH_2=y
+CONFIG_ARCH_USES_PG_ARCH_3=y
+# CONFIG_ARCH_VEXPRESS is not set
+# CONFIG_ARCH_VISCONTI is not set
+# CONFIG_ARCH_XGENE is not set
+# CONFIG_ARCH_ZYNQMP is not set
+CONFIG_ARM64_AS_HAS_MTE=y
+# CONFIG_ARM64_ERRATUM_1024718 is not set
+# CONFIG_ARM64_ERRATUM_1165522 is not set
+# CONFIG_ARM64_ERRATUM_1319367 is not set
+# CONFIG_ARM64_ERRATUM_1418040 is not set
+# CONFIG_ARM64_ERRATUM_1463225 is not set
+# CONFIG_ARM64_ERRATUM_1508412 is not set
+# CONFIG_ARM64_ERRATUM_1530923 is not set
+# CONFIG_ARM64_ERRATUM_1742098 is not set
+# CONFIG_ARM64_ERRATUM_2051678 is not set
+# CONFIG_ARM64_ERRATUM_2054223 is not set
+# CONFIG_ARM64_ERRATUM_2067961 is not set
+# CONFIG_ARM64_ERRATUM_2077057 is not set
+# CONFIG_ARM64_ERRATUM_2457168 is not set
+# CONFIG_ARM64_ERRATUM_2645198 is not set
+# CONFIG_ARM64_ERRATUM_2658417 is not set
+# CONFIG_ARM64_ERRATUM_2966298 is not set
+# CONFIG_ARM64_ERRATUM_3117295 is not set
+# CONFIG_ARM64_ERRATUM_3194386 is not set
+# CONFIG_ARM64_ERRATUM_4118414 is not set
+# CONFIG_ARM64_ERRATUM_4311569 is not set
+# CONFIG_ARM64_ERRATUM_819472 is not set
+# CONFIG_ARM64_ERRATUM_824069 is not set
+# CONFIG_ARM64_ERRATUM_826319 is not set
+# CONFIG_ARM64_ERRATUM_827319 is not set
+# CONFIG_ARM64_ERRATUM_832075 is not set
+# CONFIG_ARM64_ERRATUM_843419 is not set
+# CONFIG_ARM64_ERRATUM_845719 is not set
+# CONFIG_ARM64_GCS is not set
+CONFIG_ARM64_MTE=y
+# CONFIG_ARM64_PLATFORM_DEVICES is not set
+CONFIG_ARM64_SME=y
+# CONFIG_ARM_SBSA_WATCHDOG is not set
+# CONFIG_ARM_SMC_WATCHDOG is not set
+# CONFIG_ARM_SP805_WATCHDOG is not set
+# CONFIG_ATA is not set
+# CONFIG_ATH10K is not set
+# CONFIG_ATH11K_AHB is not set
+CONFIG_ATH11K_DEBUG=y
+CONFIG_ATH12K=y
+# CONFIG_ATL1C is not set
+CONFIG_BACKLIGHT_CLASS_DEVICE=y
+# CONFIG_BACKLIGHT_LP855X is not set
+# CONFIG_BATTERY_MAX17042 is not set
+CONFIG_BATTERY_QCOM_BATTMGR=y
+# CONFIG_BATTERY_SBS is not set
+CONFIG_BINFMT_MISC=m
+# CONFIG_BLK_DEV_MD is not set
+# CONFIG_BLK_DEV_NVME is not set
+CONFIG_BLK_DEV_RAM=y
+CONFIG_BLK_DEV_RAM_COUNT=16
+CONFIG_BLK_DEV_RAM_SIZE=8192
+CONFIG_BLK_DEV_THROTTLING=y
+# CONFIG_BNX2X is not set
+CONFIG_BOOT_CONFIG=y
+# CONFIG_BRCMFMAC is not set
+# CONFIG_BTRFS_FS is not set
+CONFIG_BT_BNEP=m
+CONFIG_BT_BNEP_MC_FILTER=y
+CONFIG_BT_BNEP_PROTO_FILTER=y
+CONFIG_BT_LE=y
+CONFIG_BT_LE_L2CAP_ECRED=y
+CONFIG_BT_RFCOMM=m
+CONFIG_BT_RFCOMM_TTY=y
+# CONFIG_CAN is not set
+# CONFIG_CAVIUM_ERRATUM_22375 is not set
+# CONFIG_CAVIUM_ERRATUM_23154 is not set
+# CONFIG_CAVIUM_ERRATUM_27456 is not set
+# CONFIG_CAVIUM_ERRATUM_30115 is not set
+# CONFIG_CAVIUM_TX2_ERRATUM_219 is not set
+CONFIG_CEC_CORE=y
+CONFIG_CFG80211=y
+CONFIG_CFS_BANDWIDTH=y
+CONFIG_CGROUP_NET_CLASSID=y
+CONFIG_CGROUP_NET_PRIO=y
+CONFIG_CHECKPOINT_RESTORE=y
+# CONFIG_CHROME_PLATFORMS is not set
+# CONFIG_CLK_GFM_LPASS_SM8250 is not set
+CONFIG_CLK_KAANAPALI_CAMCC=y
+CONFIG_CLK_KAANAPALI_DISPCC=y
+CONFIG_CLK_KAANAPALI_GPUCC=y
+CONFIG_CLK_KAANAPALI_TCSRCC=y
+CONFIG_CLK_KAANAPALI_VIDEOCC=y
+# CONFIG_CLK_QCM2290_GPUCC is not set
+# CONFIG_CLK_X1E80100_CAMCC is not set
+# CONFIG_CLK_X1E80100_DISPCC is not set
+CONFIG_CLK_X1E80100_GCC=m
+# CONFIG_CLK_X1E80100_GPUCC is not set
+# CONFIG_CLK_X1E80100_TCSRCC is not set
+# CONFIG_CLK_X1P42100_GPUCC is not set
+# CONFIG_COMMON_CLK_CS2000_CP is not set
+# CONFIG_COMMON_CLK_RS9_PCIE is not set
+# CONFIG_COMMON_CLK_VC3 is not set
+# CONFIG_COMMON_CLK_VC5 is not set
+# CONFIG_COMMON_CLK_XGENE is not set
+# CONFIG_COMMON_CLK_XLNX_CLKWZRD is not set
+CONFIG_CPU_FREQ_GOV_POWERSAVE=y
+CONFIG_CRYPTO_ACOMP2=m
+CONFIG_CRYPTO_CCM=y
+CONFIG_CRYPTO_CTR=y
+# CONFIG_CRYPTO_DEV_CCREE is not set
+CONFIG_CRYPTO_DH=y
+CONFIG_CRYPTO_ESSIV=m
+CONFIG_CRYPTO_GCM=y
+CONFIG_CRYPTO_KDF800108_CTR=y
+CONFIG_CRYPTO_KPP=y
+CONFIG_CRYPTO_KPP2=y
+CONFIG_CRYPTO_LIB_ARC4=y
+CONFIG_CRYPTO_LIB_CHACHA20POLY1305=m
+CONFIG_CRYPTO_LIB_CURVE25519=m
+CONFIG_CRYPTO_LIB_CURVE25519_GENERIC=y
+CONFIG_CRYPTO_LIB_GF128HASH=y
+CONFIG_CRYPTO_LIB_POLY1305=m
+CONFIG_CRYPTO_LIB_POLY1305_ARCH=y
+CONFIG_CRYPTO_LZ4=m
+CONFIG_CRYPTO_LZO=m
+CONFIG_CRYPTO_USER_API_HASH=m
+CONFIG_CRYPTO_USER_API_SKCIPHER=m
+CONFIG_CRYPTO_XTS=y
+CONFIG_DEFAULT_HOSTNAME="infiniti"
+CONFIG_DEFAULT_TCP_CONG="westwood"
+CONFIG_DEFAULT_WESTWOOD=y
+CONFIG_DMABUF_HEAPS_CMA=y
+CONFIG_DMABUF_HEAPS_SYSTEM=y
+CONFIG_DM_BIO_PRISON=m
+CONFIG_DM_BUFIO=m
+CONFIG_DM_CRYPT=m
+# CONFIG_DM_MIRROR is not set
+CONFIG_DM_PERSISTENT_DATA=m
+CONFIG_DM_THIN_PROVISIONING=m
+# CONFIG_DM_ZERO is not set
+CONFIG_DRM=y
+# CONFIG_DRM_ANALOGIX_ANX7625 is not set
+CONFIG_DRM_AUX_BRIDGE=y
+CONFIG_DRM_AUX_HPD_BRIDGE=y
+CONFIG_DRM_CDNS_DSI=y
+# CONFIG_DRM_CDNS_MHDP8546 is not set
+CONFIG_DRM_CLIENT_LIB=y
+CONFIG_DRM_CLIENT_LOG=y
+CONFIG_DRM_CLIENT_SELECTION=y
+CONFIG_DRM_DISPLAY_CONNECTOR=y
+CONFIG_DRM_DISPLAY_DP_AUX_BUS=y
+CONFIG_DRM_DISPLAY_DP_AUX_CEC=y
+CONFIG_DRM_DISPLAY_DP_AUX_CHARDEV=y
+CONFIG_DRM_DISPLAY_HELPER=y
+CONFIG_DRM_DRAW=y
+# CONFIG_DRM_ETNAVIV is not set
+CONFIG_DRM_EXEC=y
+CONFIG_DRM_GPUVM=y
+CONFIG_DRM_GUD=m
+# CONFIG_DRM_HDLCD is not set
+# CONFIG_DRM_HISI_HIBMC is not set
+# CONFIG_DRM_HISI_KIRIN is not set
+# CONFIG_DRM_I2C_ADV7511 is not set
+# CONFIG_DRM_I2C_NXP_TDA998X is not set
+# CONFIG_DRM_ITE_IT6263 is not set
+# CONFIG_DRM_ITE_IT66121 is not set
+CONFIG_DRM_KMS_HELPER=y
+# CONFIG_DRM_KOMEDA is not set
+# CONFIG_DRM_LIMA is not set
+# CONFIG_DRM_LONTIUM_LT8912B is not set
+# CONFIG_DRM_LONTIUM_LT9611 is not set
+# CONFIG_DRM_LONTIUM_LT9611UXC is not set
+# CONFIG_DRM_MALI_DISPLAY is not set
+CONFIG_DRM_MSM=y
+# CONFIG_DRM_MSM_DSI_10NM_PHY is not set
+# CONFIG_DRM_MSM_DSI_14NM_PHY is not set
+# CONFIG_DRM_MSM_DSI_20NM_PHY is not set
+# CONFIG_DRM_MSM_DSI_28NM_8960_PHY is not set
+# CONFIG_DRM_MSM_MDP4 is not set
+# CONFIG_DRM_MSM_MDP5 is not set
+# CONFIG_DRM_NOUVEAU is not set
+# CONFIG_DRM_NWL_MIPI_DSI is not set
+# CONFIG_DRM_PANEL_BOE_TV101WUM_NL6 is not set
+# CONFIG_DRM_PANEL_LVDS is not set
+# CONFIG_DRM_PANEL_MANTIX_MLAF057WE51 is not set
+CONFIG_DRM_PANEL_ONEPLUS15_A0020=y
+# CONFIG_DRM_PANEL_RAYDIUM_RM67191 is not set
+CONFIG_DRM_PANEL_SAMSUNG_S6E3HC3=y
+# CONFIG_DRM_PANEL_SITRONIX_ST7703 is not set
+# CONFIG_DRM_PANEL_TRULY_NT35597_WQXGA is not set
+# CONFIG_DRM_PANEL_VISIONOX_VTDR6130 is not set
+# CONFIG_DRM_PANFROST is not set
+# CONFIG_DRM_PANTHOR is not set
+# CONFIG_DRM_PARADE_PS8640 is not set
+# CONFIG_DRM_PL111 is not set
+# CONFIG_DRM_POWERVR is not set
+# CONFIG_DRM_SAMSUNG_DSIM is not set
+CONFIG_DRM_SCHED=y
+# CONFIG_DRM_SII902X is not set
+# CONFIG_DRM_SIMPLE_BRIDGE is not set
+# CONFIG_DRM_THINE_THC63LVD1024 is not set
+# CONFIG_DRM_TIDSS is not set
+# CONFIG_DRM_TI_SN65DSI83 is not set
+# CONFIG_DRM_TI_SN65DSI86 is not set
+# CONFIG_DRM_TI_TFP410 is not set
+# CONFIG_DRM_TOSHIBA_TC358767 is not set
+# CONFIG_DRM_TOSHIBA_TC358768 is not set
+CONFIG_DUMMY=m
+# CONFIG_DW_EDMA is not set
+# CONFIG_DW_WATCHDOG is not set
+CONFIG_EDAC_QCOM=y
+# CONFIG_EEPROM_AT24 is not set
+# CONFIG_EEPROM_AT25 is not set
+# CONFIG_EFI is not set
+CONFIG_EXPERT=y
+# CONFIG_EXTCON_PTN5150 is not set
+# Firmware from rootfs (/lib/firmware), not built into the kernel image.
+CONFIG_EXTRA_FIRMWARE=""
+# CONFIG_EXTRA_FIRMWARE_DIR is not set
+CONFIG_F2FS_FS=y
+CONFIG_F2FS_FS_POSIX_ACL=y
+CONFIG_F2FS_FS_XATTR=y
+CONFIG_F2FS_IOSTAT=y
+CONFIG_F2FS_STAT_FS=y
+CONFIG_FAT_DEFAULT_UTF8=y
+CONFIG_FB_SIMPLE=y
+# CONFIG_FPGA is not set
+CONFIG_FRAMEBUFFER_CONSOLE_DEFERRED_TAKEOVER=y
+CONFIG_FRAMEBUFFER_CONSOLE_LEGACY_ACCELERATION=y
+CONFIG_FRAMEBUFFER_CONSOLE_ROTATION=y
+# CONFIG_FSL_EDMA is not set
+# CONFIG_FSL_RCPM is not set
+# CONFIG_FUJITSU_ERRATUM_010001 is not set
+CONFIG_FW_CS_DSP=m
+CONFIG_FW_LOADER_COMPRESS=y
+CONFIG_FW_LOADER_COMPRESS_ZSTD=y
+# CONFIG_FW_LOADER_USER_HELPER is not set
+CONFIG_GENERIC_ADC_THERMAL=y
+# CONFIG_GNSS_MTK_SERIAL is not set
+# CONFIG_GPIO_ALTERA is not set
+# CONFIG_GPIO_DWAPB is not set
+# CONFIG_GPIO_MAX732X is not set
+# CONFIG_GPIO_MB86S7X is not set
+# CONFIG_GPIO_PCA953X is not set
+# CONFIG_GPIO_PCF857X is not set
+# CONFIG_GPIO_PL061 is not set
+CONFIG_GPIO_SYSFS=y
+CONFIG_GPIO_SYSFS_LEGACY=y
+# CONFIG_GPIO_TPIC2810 is not set
+# CONFIG_GPIO_XGENE is not set
+# CONFIG_GPIO_XILINX is not set
+CONFIG_GROUP_SCHED_BANDWIDTH=y
+CONFIG_HAVE_ARCH_KASAN_HW_TAGS=y
+CONFIG_HAVE_ARCH_KCSAN=y
+# CONFIG_HIBERNATION is not set
+CONFIG_HIDRAW=y
+CONFIG_HID_BATTERY_STRENGTH=y
+CONFIG_HID_GENERIC=m
+CONFIG_HID_SONY=m
+# CONFIG_HISILICON_ERRATUM_161600802 is not set
+# CONFIG_HISILICON_ERRATUM_162100125 is not set
+# CONFIG_HISILICON_ERRATUM_162100801 is not set
+# CONFIG_HIX5HD2_GMAC is not set
+# CONFIG_HNS3 is not set
+# CONFIG_HNS_DSAF is not set
+# CONFIG_HNS_ENET is not set
+# CONFIG_I2C_DESIGNWARE_PLATFORM is not set
+# CONFIG_I2C_MUX_PCA954x is not set
+CONFIG_I2C_QCOM_GENI=y
+# CONFIG_I2C_RK3X is not set
+# CONFIG_IGB is not set
+# CONFIG_IGBVF is not set
+CONFIG_INPUT_JOYDEV=m
+CONFIG_INPUT_UINPUT=m
+# CONFIG_INTERCONNECT_QCOM_MILOS is not set
+# CONFIG_INTERCONNECT_QCOM_MSM8916 is not set
+# CONFIG_INTERCONNECT_QCOM_MSM8953 is not set
+# CONFIG_INTERCONNECT_QCOM_MSM8996 is not set
+CONFIG_INTERCONNECT_QCOM_OSM_L3=y
+# CONFIG_INTERCONNECT_QCOM_QCM2290 is not set
+# CONFIG_INTERCONNECT_QCOM_QCS404 is not set
+# CONFIG_INTERCONNECT_QCOM_QCS615 is not set
+# CONFIG_INTERCONNECT_QCOM_QCS8300 is not set
+# CONFIG_INTERCONNECT_QCOM_QDU1000 is not set
+# CONFIG_INTERCONNECT_QCOM_SA8775P is not set
+# CONFIG_INTERCONNECT_QCOM_SC7180 is not set
+# CONFIG_INTERCONNECT_QCOM_SC7280 is not set
+# CONFIG_INTERCONNECT_QCOM_SC8180X is not set
+# CONFIG_INTERCONNECT_QCOM_SC8280XP is not set
+# CONFIG_INTERCONNECT_QCOM_SDM845 is not set
+# CONFIG_INTERCONNECT_QCOM_SDX75 is not set
+# CONFIG_INTERCONNECT_QCOM_SM6115 is not set
+# CONFIG_INTERCONNECT_QCOM_SM6350 is not set
+# CONFIG_INTERCONNECT_QCOM_SM8150 is not set
+# CONFIG_INTERCONNECT_QCOM_SM8250 is not set
+# CONFIG_INTERCONNECT_QCOM_SM8350 is not set
+# CONFIG_INTERCONNECT_QCOM_SM8550 is not set
+# CONFIG_INTERCONNECT_QCOM_SM8650 is not set
+# CONFIG_INTERCONNECT_QCOM_SM8750 is not set
+# CONFIG_INTERCONNECT_QCOM_X1E80100 is not set
+# CONFIG_IPQ_APSS_5424 is not set
+# CONFIG_IPQ_APSS_6018 is not set
+# CONFIG_IPQ_GCC_5018 is not set
+# CONFIG_IPQ_GCC_5332 is not set
+# CONFIG_IPQ_GCC_5424 is not set
+# CONFIG_IPQ_GCC_6018 is not set
+# CONFIG_IPQ_GCC_8074 is not set
+# CONFIG_IPQ_GCC_9574 is not set
+CONFIG_IPVLAN=m
+CONFIG_IPVLAN_L3S=y
+CONFIG_IP_VS_NFCT=y
+CONFIG_IP_VS_PROTO_TCP=y
+CONFIG_IP_VS_PROTO_UDP=y
+CONFIG_IP_VS_RR=m
+# CONFIG_IR_GPIO_CIR is not set
+CONFIG_IR_SPI=y
+# CONFIG_KEYBOARD_ADC is not set
+CONFIG_KEY_DH_OPERATIONS=y
+CONFIG_LEDS_CLASS_FLASH=y
+CONFIG_LEDS_QCOM_FLASH=y
+CONFIG_LEDS_TRIGGER_ACTIVITY=y
+CONFIG_LEDS_TRIGGER_BACKLIGHT=y
+CONFIG_LEDS_TRIGGER_ONESHOT=y
+CONFIG_LIRC=y
+CONFIG_LOCALVERSION="-oneplus-infiniti"
+# CONFIG_LOCALVERSION_AUTO is not set
+CONFIG_LZ4_COMPRESS=m
+CONFIG_LZO_COMPRESS=m
+CONFIG_MAC80211=y
+# CONFIG_MACB is not set
+# CONFIG_MAX9611 is not set
+# CONFIG_MDIO_BUS_MUX_MMIOREG is not set
+# CONFIG_MDIO_BUS_MUX_MULTIPLEXER is not set
+# CONFIG_MEDIA_ANALOG_TV_SUPPORT is not set
+# CONFIG_MEDIA_DIGITAL_TV_SUPPORT is not set
+# CONFIG_MEDIA_SDR_SUPPORT is not set
+# CONFIG_MEGARAID_SAS is not set
+# CONFIG_MEMORY_HOTPLUG is not set
+# CONFIG_MFD_ADP5585 is not set
+# CONFIG_MFD_AXP20X_I2C is not set
+# CONFIG_MFD_BD9571MWV is not set
+# CONFIG_MFD_DA9062 is not set
+# CONFIG_MFD_HI6421_PMIC is not set
+# CONFIG_MFD_MAX77620 is not set
+# CONFIG_MFD_MAX77759 is not set
+# CONFIG_MFD_MT6360 is not set
+# CONFIG_MFD_MT6397 is not set
+# CONFIG_MFD_RK8XX_I2C is not set
+# CONFIG_MFD_RK8XX_SPI is not set
+# CONFIG_MFD_ROHM_BD718XX is not set
+# CONFIG_MFD_SEC_I2C is not set
+# CONFIG_MFD_TI_LP873X is not set
+# CONFIG_MFD_TPS65219 is not set
+# CONFIG_MFD_TPS6594_I2C is not set
+# CONFIG_MFD_WCD934X is not set
+# CONFIG_MFD_WM8994 is not set
+CONFIG_MHI_BUS=y
+# CONFIG_MICREL_PHY is not set
+# CONFIG_MICROSEMI_PHY is not set
+# CONFIG_MISC_RP1 is not set
+# CONFIG_MLX4_EN is not set
+# CONFIG_MLX5_CORE is not set
+# CONFIG_MMC_DW_EXYNOS is not set
+# CONFIG_MMC_DW_HI3798CV200 is not set
+# CONFIG_MMC_DW_K3 is not set
+# CONFIG_MMC_MTK is not set
+# CONFIG_MMC_SDHCI_CADENCE is not set
+# CONFIG_MMC_SDHCI_F_SDH30 is not set
+# CONFIG_MMC_SDHCI_OF_ARASAN is not set
+# CONFIG_MMC_SDHCI_XENON is not set
+# CONFIG_MOUSE_ELAN_I2C is not set
+# CONFIG_MPL3115 is not set
+# CONFIG_MSM_GCC_8916 is not set
+# CONFIG_MSM_GCC_8953 is not set
+# CONFIG_MSM_GCC_8994 is not set
+# CONFIG_MSM_GCC_8996 is not set
+# CONFIG_MSM_GCC_8998 is not set
+# CONFIG_MSM_MMCC_8994 is not set
+# CONFIG_MSM_MMCC_8996 is not set
+# CONFIG_MSM_MMCC_8998 is not set
+# CONFIG_MT7921E is not set
+# CONFIG_MTD is not set
+# CONFIG_MV_XOR_V2 is not set
+CONFIG_MXM_DOWN=y
+CONFIG_MXM_UP=y
+CONFIG_NETFILTER_NETLINK=y
+CONFIG_NETFILTER_XT_MATCH_COMMENT=m
+CONFIG_NETLINK_DIAG=m
+# CONFIG_NET_9P is not set
+# CONFIG_NET_CLS_ACT is not set
+# CONFIG_NET_CLS_BASIC is not set
+CONFIG_NET_CLS_CGROUP=m
+# CONFIG_NET_CLS_FLOWER is not set
+# CONFIG_NET_DSA is not set
+CONFIG_NET_L3_MASTER_DEV=y
+# CONFIG_NET_SCH_CBS is not set
+# CONFIG_NET_SCH_ETF is not set
+CONFIG_NET_SCH_HTB=y
+# CONFIG_NET_SCH_MQPRIO is not set
+CONFIG_NET_SCH_MULTIQ=y
+CONFIG_NET_SCH_PRIO=y
+# CONFIG_NET_SCH_TAPRIO is not set
+CONFIG_NET_UDP_TUNNEL=m
+# CONFIG_NET_VENDOR_SOCIONEXT is not set
+CONFIG_NFT_CT=m
+CONFIG_NFT_LIMIT=m
+CONFIG_NFT_LOG=m
+CONFIG_NFT_MASQ=m
+CONFIG_NFT_NAT=m
+CONFIG_NFT_REJECT=m
+CONFIG_NFT_REJECT_INET=m
+CONFIG_NFT_REJECT_IPV4=m
+CONFIG_NFT_REJECT_IPV6=m
+CONFIG_NF_NAT=m
+CONFIG_NF_NAT_MASQUERADE=y
+CONFIG_NF_REJECT_IPV4=m
+CONFIG_NF_REJECT_IPV6=m
+CONFIG_NF_TABLES=m
+CONFIG_NF_TABLES_INET=y
+CONFIG_NF_TABLES_IPV4=y
+CONFIG_NF_TABLES_IPV6=y
+CONFIG_NLS_DEFAULT="utf8"
+CONFIG_NLS_UTF8=y
+# CONFIG_NOP_USB_XCEIV is not set
+CONFIG_NR_CPUS=8
+# CONFIG_NUMA is not set
+# CONFIG_NVIDIA_CARMEL_CNP_ERRATUM is not set
+CONFIG_NVMEM_REBOOT_MODE=y
+# CONFIG_NVMEM_RMEM is not set
+CONFIG_PACKET_DIAG=y
+# CONFIG_PCIE_ALTERA is not set
+# CONFIG_PCIE_KIRIN is not set
+# CONFIG_PCI_HISI is not set
+# CONFIG_PCI_HOST_THUNDER_ECAM is not set
+# CONFIG_PCI_HOST_THUNDER_PEM is not set
+# CONFIG_PCI_MESON is not set
+# CONFIG_PCI_PASID is not set
+CONFIG_PCI_PWRCTRL_PWRSEQ=y
+# CONFIG_PCI_XGENE is not set
+CONFIG_PHY_PACKAGE=m
+CONFIG_PHY_QCOM_EDP=y
+CONFIG_PHY_QCOM_EUSB2_REPEATER=y
+CONFIG_PHY_QCOM_M31_EUSB=y
+CONFIG_PHY_QCOM_M31_USB=y
+CONFIG_PHY_QCOM_PCIE2=y
+CONFIG_PHY_QCOM_QMP=y
+CONFIG_PHY_QCOM_QMP_COMBO=y
+CONFIG_PHY_QCOM_QMP_PCIE=y
+CONFIG_PHY_QCOM_QMP_PCIE_8996=y
+CONFIG_PHY_QCOM_QMP_UFS=y
+CONFIG_PHY_QCOM_QMP_USB=y
+CONFIG_PHY_QCOM_QUSB2=y
+CONFIG_PHY_QCOM_USB_HS=y
+CONFIG_PHY_QCOM_USB_HSIC=y
+CONFIG_PHY_QCOM_USB_HS_28NM=y
+CONFIG_PHY_QCOM_USB_SNPS_FEMTO_V2=y
+CONFIG_PHY_QCOM_USB_SS=y
+# CONFIG_PINCTRL_IPQ5018 is not set
+# CONFIG_PINCTRL_IPQ5332 is not set
+# CONFIG_PINCTRL_IPQ5424 is not set
+# CONFIG_PINCTRL_IPQ6018 is not set
+# CONFIG_PINCTRL_IPQ8074 is not set
+# CONFIG_PINCTRL_IPQ9574 is not set
+CONFIG_PINCTRL_LPASS_LPI=y
+# CONFIG_PINCTRL_MILOS is not set
+# CONFIG_PINCTRL_MSM8916 is not set
+# CONFIG_PINCTRL_MSM8953 is not set
+# CONFIG_PINCTRL_MSM8976 is not set
+# CONFIG_PINCTRL_MSM8994 is not set
+# CONFIG_PINCTRL_MSM8996 is not set
+# CONFIG_PINCTRL_MSM8998 is not set
+# CONFIG_PINCTRL_QCM2290 is not set
+# CONFIG_PINCTRL_QCS404 is not set
+# CONFIG_PINCTRL_QCS615 is not set
+# CONFIG_PINCTRL_QCS8300 is not set
+# CONFIG_PINCTRL_QDU1000 is not set
+# CONFIG_PINCTRL_SA8775P is not set
+# CONFIG_PINCTRL_SC7180 is not set
+# CONFIG_PINCTRL_SC7280 is not set
+# CONFIG_PINCTRL_SC7280_LPASS_LPI is not set
+# CONFIG_PINCTRL_SC8180X is not set
+# CONFIG_PINCTRL_SC8280XP is not set
+# CONFIG_PINCTRL_SC8280XP_LPASS_LPI is not set
+# CONFIG_PINCTRL_SDM660 is not set
+# CONFIG_PINCTRL_SDM670 is not set
+# CONFIG_PINCTRL_SDM845 is not set
+# CONFIG_PINCTRL_SDX75 is not set
+# CONFIG_PINCTRL_SM4450 is not set
+# CONFIG_PINCTRL_SM6115 is not set
+# CONFIG_PINCTRL_SM6115_LPASS_LPI is not set
+# CONFIG_PINCTRL_SM6125 is not set
+# CONFIG_PINCTRL_SM6350 is not set
+# CONFIG_PINCTRL_SM6375 is not set
+# CONFIG_PINCTRL_SM8150 is not set
+# CONFIG_PINCTRL_SM8250 is not set
+# CONFIG_PINCTRL_SM8250_LPASS_LPI is not set
+# CONFIG_PINCTRL_SM8350 is not set
+CONFIG_PINCTRL_SM8450_LPASS_LPI=y
+# CONFIG_PINCTRL_SM8550 is not set
+# CONFIG_PINCTRL_SM8550_LPASS_LPI is not set
+# CONFIG_PINCTRL_SM8650 is not set
+# CONFIG_PINCTRL_SM8650_LPASS_LPI is not set
+# CONFIG_PINCTRL_SM8750 is not set
+# CONFIG_PINCTRL_SX150X is not set
+# CONFIG_PINCTRL_X1E80100 is not set
+CONFIG_PKCS8_PRIVATE_KEY_PARSER=y
+# CONFIG_PL330_DMA is not set
+CONFIG_PM_AUTOSLEEP=y
+CONFIG_PM_WAKELOCKS=y
+CONFIG_PM_WAKELOCKS_GC=y
+CONFIG_PM_WAKELOCKS_LIMIT=100
+CONFIG_POWER_RESET_QCOM_PON=y
+# CONFIG_POWER_RESET_SYSCON is not set
+# CONFIG_POWER_RESET_XGENE is not set
+CONFIG_POWER_SEQUENCING_QCOM_WCN=y
+CONFIG_PROC_CHILDREN=y
+CONFIG_PSI=y
+CONFIG_PSTORE_CONSOLE=y
+CONFIG_PSTORE_PMSG=y
+CONFIG_PSTORE_RAM=y
+# CONFIG_QCM_DISPCC_2290 is not set
+# CONFIG_QCM_GCC_2290 is not set
+# CONFIG_QCOM_A53PLL is not set
+CONFIG_QCOM_APR=y
+# CONFIG_QCOM_CLK_APCC_MSM8996 is not set
+# CONFIG_QCOM_CLK_APCS_MSM8916 is not set
+# CONFIG_QCOM_FALKOR_ERRATUM_1003 is not set
+# CONFIG_QCOM_FALKOR_ERRATUM_1009 is not set
+# CONFIG_QCOM_FALKOR_ERRATUM_E1041 is not set
+CONFIG_QCOM_GPI_DMA=y
+CONFIG_QCOM_INLINE_CRYPTO_ENGINE=y
+CONFIG_QCOM_LLCC=y
+CONFIG_QCOM_MDT_LOADER=y
+CONFIG_QCOM_OCMEM=y
+CONFIG_QCOM_PDR_HELPERS=y
+CONFIG_QCOM_PDR_MSG=y
+CONFIG_QCOM_PMIC_GLINK=y
+# CONFIG_QCOM_QDF2400_ERRATUM_0065 is not set
+CONFIG_QCOM_QMI_HELPERS=y
+CONFIG_QCOM_RMTFS_MEM=y
+CONFIG_QCOM_SOCINFO=y
+CONFIG_QCOM_SPMI_ADC5=y
+CONFIG_QCOM_SPMI_ADC_TM5=y
+CONFIG_QCOM_SPMI_TEMP_ALARM=y
+CONFIG_QCOM_UBWC_CONFIG=y
+CONFIG_QCOM_VADC_COMMON=y
+# CONFIG_QCS_CAMCC_615 is not set
+# CONFIG_QCS_DISPCC_615 is not set
+# CONFIG_QCS_GCC_404 is not set
+# CONFIG_QCS_GCC_615 is not set
+# CONFIG_QCS_GCC_8300 is not set
+# CONFIG_QCS_GPUCC_615 is not set
+# CONFIG_QCS_VIDEOCC_615 is not set
+# CONFIG_QDU_GCC_1000 is not set
+CONFIG_QFMT_V2=y
+CONFIG_QRTR=y
+CONFIG_QRTR_MHI=y
+CONFIG_QRTR_SMD=y
+CONFIG_QRTR_TUN=y
+CONFIG_QUOTA_TREE=y
+CONFIG_RC_CORE=y
+# CONFIG_RC_DECODERS is not set
+CONFIG_REALTEK_PHY=m
+CONFIG_REED_SOLOMON=y
+CONFIG_REGMAP_SPI=m
+# CONFIG_REGULATOR_FAN53555 is not set
+# CONFIG_REGULATOR_MAX8973 is not set
+# CONFIG_REGULATOR_MP8859 is not set
+# CONFIG_REGULATOR_MT6315 is not set
+# CONFIG_REGULATOR_PCA9450 is not set
+# CONFIG_REGULATOR_PF8X00 is not set
+# CONFIG_REGULATOR_PFUZE100 is not set
+# CONFIG_REGULATOR_RAA215300 is not set
+# CONFIG_REGULATOR_TPS65132 is not set
+# CONFIG_REGULATOR_VCTRL is not set
+CONFIG_REMOTEPROC_CDEV=y
+# CONFIG_RESET_SIMPLE is not set
+CONFIG_RFKILL=y
+# CONFIG_ROCKCHIP_ERRATUM_3568002 is not set
+# CONFIG_ROCKCHIP_ERRATUM_3588001 is not set
+CONFIG_RPMSG_CHAR=y
+CONFIG_RPMSG_QCOM_GLINK_SMEM=y
+# CONFIG_RTC_DRV_BQ32K is not set
+# CONFIG_RTC_DRV_DS1307 is not set
+# CONFIG_RTC_DRV_DS3232 is not set
+# CONFIG_RTC_DRV_HYM8563 is not set
+# CONFIG_RTC_DRV_IMX_BBM_SCMI is not set
+# CONFIG_RTC_DRV_ISL1208 is not set
+# CONFIG_RTC_DRV_M41T80 is not set
+# CONFIG_RTC_DRV_PCF2127 is not set
+# CONFIG_RTC_DRV_PCF85063 is not set
+# CONFIG_RTC_DRV_PCF85363 is not set
+# CONFIG_RTC_DRV_PCF8563 is not set
+# CONFIG_RTC_DRV_PL031 is not set
+# CONFIG_RTC_DRV_RV3028 is not set
+# CONFIG_RTC_DRV_RV8803 is not set
+# CONFIG_RTC_DRV_RX8581 is not set
+# CONFIG_RTC_DRV_ZYNQMP is not set
+# CONFIG_SA_CAMCC_8775P is not set
+# CONFIG_SA_DISPCC_8775P is not set
+# CONFIG_SA_GCC_8775P is not set
+# CONFIG_SA_GPUCC_8775P is not set
+# CONFIG_SA_VIDEOCC_8775P is not set
+# CONFIG_SCHED_SMT is not set
+# CONFIG_SCSI_MPT3SAS is not set
+CONFIG_SCSI_SCAN_ASYNC=y
+CONFIG_SCSI_UFS_CDNS_PLATFORM=y
+CONFIG_SCSI_UFS_QCOM=y
+# CONFIG_SC_CAMCC_7280 is not set
+# CONFIG_SC_CAMCC_8280XP is not set
+# CONFIG_SC_DISPCC_7280 is not set
+# CONFIG_SC_DISPCC_8280XP is not set
+# CONFIG_SC_GCC_7180 is not set
+# CONFIG_SC_GCC_7280 is not set
+# CONFIG_SC_GCC_8180X is not set
+# CONFIG_SC_GCC_8280XP is not set
+# CONFIG_SC_GPUCC_7280 is not set
+# CONFIG_SC_GPUCC_8280XP is not set
+# CONFIG_SC_LPASSCC_8280XP is not set
+# CONFIG_SC_LPASS_CORECC_7280 is not set
+# CONFIG_SC_VIDEOCC_7280 is not set
+# CONFIG_SDM_CAMCC_845 is not set
+# CONFIG_SDM_DISPCC_845 is not set
+# CONFIG_SDM_GCC_845 is not set
+# CONFIG_SDM_GPUCC_845 is not set
+# CONFIG_SDM_LPASSCC_845 is not set
+# CONFIG_SDM_VIDEOCC_845 is not set
+# CONFIG_SDX_GCC_75 is not set
+CONFIG_SECURITY_LANDLOCK=y
+CONFIG_SECURITY_NETWORK=y
+CONFIG_SECURITY_PATH=y
+# CONFIG_SENSORS_ARM_SCPI is not set
+# CONFIG_SENSORS_INA2XX is not set
+# CONFIG_SENSORS_INA3221 is not set
+# CONFIG_SENSORS_ISL29018 is not set
+# CONFIG_SENSORS_LM75 is not set
+# CONFIG_SENSORS_LM90 is not set
+# CONFIG_SENSORS_PWM_FAN is not set
+# CONFIG_SERIAL_FSL_LINFLEXUART is not set
+# CONFIG_SERIAL_FSL_LPUART is not set
+# CONFIG_SERIAL_XILINX_PS_UART is not set
+CONFIG_SLIMBUS=y
+# CONFIG_SMC91X is not set
+# CONFIG_SM_CAMCC_6350 is not set
+# CONFIG_SM_CAMCC_8250 is not set
+CONFIG_SM_CAMCC_8450=y
+# CONFIG_SM_CAMCC_8550 is not set
+# CONFIG_SM_CAMCC_8650 is not set
+# CONFIG_SM_CAMCC_MILOS is not set
+CONFIG_SM_DISPCC_8450=y
+# CONFIG_SM_DISPCC_8750 is not set
+# CONFIG_SM_GCC_4450 is not set
+# CONFIG_SM_GCC_6115 is not set
+# CONFIG_SM_GCC_6350 is not set
+# CONFIG_SM_GCC_8150 is not set
+# CONFIG_SM_GCC_8250 is not set
+# CONFIG_SM_GCC_8350 is not set
+# CONFIG_SM_GCC_8550 is not set
+# CONFIG_SM_GCC_8650 is not set
+CONFIG_SM_GCC_8750=m
+# CONFIG_SM_GCC_MILOS is not set
+# CONFIG_SM_GPUCC_6115 is not set
+# CONFIG_SM_GPUCC_6350 is not set
+# CONFIG_SM_GPUCC_8150 is not set
+# CONFIG_SM_GPUCC_8250 is not set
+# CONFIG_SM_GPUCC_8350 is not set
+CONFIG_SM_GPUCC_8450=y
+# CONFIG_SM_GPUCC_8550 is not set
+# CONFIG_SM_GPUCC_8650 is not set
+# CONFIG_SM_GPUCC_MILOS is not set
+# CONFIG_SM_TCSRCC_8550 is not set
+# CONFIG_SM_TCSRCC_8650 is not set
+# CONFIG_SM_TCSRCC_8750 is not set
+# CONFIG_SM_VIDEOCC_6350 is not set
+# CONFIG_SM_VIDEOCC_8250 is not set
+CONFIG_SM_VIDEOCC_8450=y
+# CONFIG_SM_VIDEOCC_8550 is not set
+# CONFIG_SM_VIDEOCC_MILOS is not set
+# CONFIG_SND_AUDIO_GRAPH_CARD is not set
+# CONFIG_SND_SIMPLE_CARD is not set
+# CONFIG_SND_SOC_AK4613 is not set
+# CONFIG_SND_SOC_CROS_EC_CODEC is not set
+CONFIG_SND_SOC_CS35L41=m
+CONFIG_SND_SOC_CS35L41_I2C=m
+CONFIG_SND_SOC_CS35L41_LIB=m
+# CONFIG_SND_SOC_DA7213 is not set
+# CONFIG_SND_SOC_ES7134 is not set
+# CONFIG_SND_SOC_ES7241 is not set
+# CONFIG_SND_SOC_ES8316 is not set
+# CONFIG_SND_SOC_FSL_ASRC is not set
+# CONFIG_SND_SOC_FSL_AUDMIX is not set
+# CONFIG_SND_SOC_FSL_MICFIL is not set
+# CONFIG_SND_SOC_FSL_SAI is not set
+# CONFIG_SND_SOC_FSL_SPDIF is not set
+# CONFIG_SND_SOC_GTM601 is not set
+# CONFIG_SND_SOC_MSM8916_WCD_ANALOG is not set
+# CONFIG_SND_SOC_MSM8916_WCD_DIGITAL is not set
+# CONFIG_SND_SOC_MT6358 is not set
+# CONFIG_SND_SOC_NAU8822 is not set
+# CONFIG_SND_SOC_PCM3168A_I2C is not set
+# CONFIG_SND_SOC_RT5640 is not set
+# CONFIG_SND_SOC_RT5659 is not set
+# CONFIG_SND_SOC_SGTL5000 is not set
+# CONFIG_SND_SOC_TAS2552 is not set
+# CONFIG_SND_SOC_TAS571X is not set
+CONFIG_SND_SOC_TFA9872=m
+# CONFIG_SND_SOC_TLV320AIC31XX is not set
+# CONFIG_SND_SOC_TS3A227E is not set
+# CONFIG_SND_SOC_WCD9335 is not set
+# CONFIG_SND_SOC_WM8524 is not set
+# CONFIG_SND_SOC_WM8904 is not set
+# CONFIG_SND_SOC_WM8960 is not set
+# CONFIG_SND_SOC_WM8962 is not set
+# CONFIG_SND_SOC_WM8978 is not set
+CONFIG_SND_SOC_WM_ADSP=m
+# CONFIG_SND_SOC_WSA881X is not set
+# CONFIG_SOCIONEXT_SYNQUACER_PREITS is not set
+CONFIG_SONY_FF=y
+# CONFIG_SPI_CADENCE_QUADSPI is not set
+# CONFIG_SPI_DESIGNWARE is not set
+# CONFIG_SPI_PL022 is not set
+CONFIG_SPI_QCOM_GENI=y
+# CONFIG_SPI_XILINX is not set
+# CONFIG_SPI_ZYNQMP_GQSPI is not set
+CONFIG_SQUASHFS_LZ4=y
+CONFIG_SQUASHFS_LZO=y
+CONFIG_SQUASHFS_XATTR=y
+CONFIG_SQUASHFS_XZ=y
+# CONFIG_SRAM is not set
+CONFIG_STACKPROTECTOR_PER_TASK=y
+# CONFIG_STAGING_MEDIA is not set
+# CONFIG_STMMAC_ETH is not set
+# CONFIG_SURFACE_PLATFORMS is not set
+CONFIG_SYN_COOKIES=y
+# CONFIG_TCG_TPM is not set
+CONFIG_TCP_CONG_ADVANCED=y
+CONFIG_TCP_CONG_BIC=m
+CONFIG_TCP_CONG_HTCP=m
+CONFIG_TCP_CONG_WESTWOOD=y
+# CONFIG_TOUCHSCREEN_ATMEL_MXT is not set
+# CONFIG_TOUCHSCREEN_EDT_FT5X06 is not set
+# CONFIG_TOUCHSCREEN_ELAN is not set
+# CONFIG_TOUCHSCREEN_GOODIX is not set
+CONFIG_TOUCHSCREEN_SYNAPTICS_TCM=y
+CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_ONCELL=y
+CONFIG_TOUCHSCREEN_SYNAPTICS_TCM_SPI=y
+CONFIG_TYPEC=y
+CONFIG_TYPEC_DP_ALTMODE=y
+# CONFIG_TYPEC_FUSB302 is not set
+# CONFIG_TYPEC_HD3SS3220 is not set
+CONFIG_TYPEC_MUX_FSA4480=y
+CONFIG_TYPEC_MUX_GPIO_SBU=y
+CONFIG_TYPEC_MUX_NB7VPQ904M=y
+CONFIG_TYPEC_QCOM_PMIC=y
+CONFIG_TYPEC_TCPM=y
+CONFIG_TYPEC_UCSI=y
+# CONFIG_UACCE is not set
+CONFIG_UCSI_PMIC_GLINK=y
+CONFIG_UEVENT_HELPER=y
+CONFIG_UEVENT_HELPER_PATH=""
+CONFIG_UHID=m
+CONFIG_UNIX_DIAG=y
+CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
+# CONFIG_USB_CHIPIDEA is not set
+CONFIG_USB_CONFIGFS=y
+CONFIG_USB_CONFIGFS_F_HID=y
+# CONFIG_USB_CONN_GPIO is not set
+# CONFIG_USB_DWC2 is not set
+CONFIG_USB_F_ACM=y
+CONFIG_USB_F_ECM=y
+CONFIG_USB_F_EEM=y
+CONFIG_USB_F_FS=y
+CONFIG_USB_F_HID=y
+CONFIG_USB_F_MASS_STORAGE=y
+CONFIG_USB_F_NCM=y
+CONFIG_USB_F_OBEX=y
+CONFIG_USB_F_RNDIS=y
+CONFIG_USB_F_SERIAL=y
+CONFIG_USB_F_SUBSET=y
+CONFIG_USB_HID=m
+# CONFIG_USB_HSIC_USB3503 is not set
+# CONFIG_USB_ISP1760 is not set
+CONFIG_USB_LIBCOMPOSITE=y
+# CONFIG_USB_MUSB_HDRC is not set
+# CONFIG_USB_NET_PLUSB is not set
+CONFIG_USB_U_ETHER=y
+CONFIG_USB_U_SERIAL=y
+# CONFIG_USB_XHCI_PCI_RENESAS is not set
+CONFIG_U_SERIAL_CONSOLE=y
+# CONFIG_VIDEO_IMX219 is not set
+# CONFIG_VIDEO_IMX412 is not set
+# CONFIG_VIDEO_OV5640 is not set
+# CONFIG_VIDEO_OV5645 is not set
+# CONFIG_VIRTUALIZATION is not set
+# CONFIG_VITESSE_PHY is not set
+CONFIG_VXLAN=m
+# CONFIG_WCN36XX is not set
+CONFIG_WIREGUARD=m
+# CONFIG_WL18XX is not set
+# CONFIG_WLAN_VENDOR_INTEL is not set
+# CONFIG_WLAN_VENDOR_MARVELL is not set
+# CONFIG_WLAN_VENDOR_MICROCHIP is not set
+# CONFIG_WLAN_VENDOR_PURELIFI is not set
+# CONFIG_WLAN_VENDOR_QUANTENNA is not set
+# CONFIG_WLAN_VENDOR_RALINK is not set
+# CONFIG_WLCORE is not set
+# CONFIG_XILINX_DMA is not set
+# CONFIG_XILINX_SDFEC is not set
+# CONFIG_XILINX_VCU is not set
+# CONFIG_XILINX_ZYNQMP_DMA is not set
+# CONFIG_XILINX_ZYNQMP_DPDMA is not set
+CONFIG_ZRAM=m
+CONFIG_ZRAM_BACKEND_FORCE_LZO=y
+CONFIG_ZRAM_BACKEND_LZO=y
+CONFIG_ZRAM_DEF_COMP="lzo-rle"
+CONFIG_ZRAM_DEF_COMP_LZORLE=y
+CONFIG_ZRAM_MEMORY_TRACKING=y
+CONFIG_ZRAM_TRACK_ENTRY_ACTIME=y
+CONFIG_ZSMALLOC=m
+CONFIG_ZSMALLOC_CHAIN_SIZE=8
+CONFIG_ZSMALLOC_STAT=y
-- 
2.43.0

